README.md

```diff
@@ -14,18 +14,12 @@ Once you've created something amazing with Open MCT, showcase your work in our G
 Building and running Open MCT in your local dev environment is very easy. Be sure you have [Git](https://git-scm.com/downloads) and [Node.js](https://nodejs.org/) installed, then follow the directions below. Need additional information? Check out the [Getting Started](https://nasa.github.io/openmct/getting-started/) page on our website.
 (These instructions assume you are installing as a non-root user; developers have [reported issues](https://github.com/nasa/openmct/issues/1151) running these steps with root privileges.)
 
-1. Clone the source code
-
- `git clone https://github.com/nasa/openmct.git`
+1. Clone the source code `git clone https://github.com/nasa/openmct.git`
```

Comment:
This displays the code inline which also doesn't look super clean:

![image](https://github.com/nasa/openmct/assets/9259993/3c222116-c4ce-46c6-9def-642cda614bb4)


It might look better to wrap the commands in code blocks instead? Like this:

1. Clone the source code:
\`\`\`
git clone https://github.com/nasa/openmct.git
\`\`\`

Which would display as such:

![image](https://github.com/nasa/openmct/assets/9259993/6dfeb68f-0cee-431a-a9b0-526a869fe177)

Reply:
I was using chrome and the preview feature:

https://github.com/nasa/openmct/blob/28788490197bb2e75cba2c923caf344f3d4ad06b/README.md